readthedocs/proxito/middleware.py

```diff
@@ -278,9 +279,19 @@ def process_request(self, request):  # noqa
 
         return None
 
+    def add_hosting_integrations_headers(self, request, response):
+        project_slug = getattr(request, "path_project_slug", "")
+        if project_slug:
+            project = Project.objects.get(slug=project_slug)
+            if project.has_feature(Feature.HOSTING_INTEGRATIONS):
+                response["X-RTD-Hosting-Integrations"] = "true"
+            else:
+                response["X-RTD-Hosting-Integrations"] = "false"
```

Comment:
```suggestion
            if project.has_feature(Feature.HOSTING_INTEGRATIONS):
                response["X-RTD-Hosting-Integrations"] = "true"
```

I think we just don't want the header when it's off?

Reply:
Makes sense 👍🏼